cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/BatchFileFilter.java

```diff
@@ -85,7 +90,7 @@ public boolean accept(Path path) {
       LOG.warn("Couldn't parse file: " + path.getName());
       fileDate = prevMinute;
```

Comment:
It seems like if `fileDate` is equal to `prevMinute`, then `toRead(fileDate)` will return true.
So, if a `ParseException` does happen, it will definitely read the file.
Am I understanding that correctly?